lms/models/oauth_state.py

```diff
@@ -28,7 +28,19 @@ def find_or_create_from_user(session, state, user, lti_params):
 
 
 def find_user_from_state(session, state):
-    state = find_by_state(session, state)
+    state = _find_by_state(session, state)
     if state is None:
         return None
     return session.query(User).filter(User.id == state.user_id).one_or_none()
+
+
+def find_lti_params(session, state):
+    oauth_state = _find_by_state(session, state)
+    if oauth_state is None:
+        return None
+    return json.loads(oauth_state.lti_params)
```

Comment:
All these `find_*` functions should be classmethods on the `OauthState` class itself. And getting the LTI params from an `OauthState` object as a dict rather than a JSON string should just be a `my_oauth_state.lti_params` property. But these aren't trivial to fix, and I'm not sure stashing LTI params in the DB should be done at all anyway, so I'm following the existing pattern here for now.

Reply:
Might be helpful to put a TODO or similar comment in this function?